### ISSUE TITLE
HOI-A dataset

### ISSUE THREAD

**Issue by XiangjiBU** [YueLiao/PPDM#14]
Thanks for sharing! but, could you share your HOI-A dataset mentioned in you paper? 
here is my email:  xjbu2008@163.com

**Comment by YueLiao** [comment]
I am sorry for the delay of the HOI-A dataset due to the COVID-2019. We are still refining the HOI-A dataset now. You can use the HOI-W dataset first, which is the coarse version of HOI-A. More about the HOI-A dataset and HOI-W dataset can be seen in #8.

**Comment by XiangjiBU** [comment]
got it, thanks!  By the way, which label tool you used to build this dataset? because I want to build my own data

**Comment by YueLiao** [comment]
Our HOI-A/HOI-W dataset is annotated by the SenseTime Ltd Group, thus I cannot get the label tool. We can only provide the annotated process. We first annotated the human and human-related bounding-boxes for each image. Then, we visualized all bounding-boxes (each bounding-box will be given a unique ID) for each image. Based on the visualization, if ith and jth exist relationship type k, the <I,j,k> will be recorded.
